sphinx_rtd_theme/layout.html

```diff
@@ -28,16 +28,47 @@
     <link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html"/>
   {% endif %}
 
-  {# CSS #}
-
-  {# OPENSEARCH #}
+  {# Javascript -- Keepin in head #}
   {% if not embedded %}
-    {% if use_opensearch %}
-      <link rel="search" type="application/opensearchdescription+xml" title="{% trans docstitle=docstitle|e %}Search within {{ docstitle }}{% endtrans %}" href="{{ pathto('_static/opensearch.xml', 1) }}"/>
+    <script type="text/javascript">
+        var DOCUMENTATION_OPTIONS = {
+            URL_ROOT:'{{ url_root }}',
+            VERSION:'{{ release|e }}',
+            LANGUAGE:'{{ language }}',
+            COLLAPSE_INDEX:false,
+            FILE_SUFFIX:'{{ '' if no_search_suffix else file_suffix }}',
+            HAS_SOURCE:  {{ has_source|lower }},
+            SOURCELINK_SUFFIX: '{{ sourcelink_suffix }}'
+        };
+    </script>
+    {%- for scriptfile in script_files %}
+      <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
+    {%- endfor %}
+
+    {# RTD hosts this file, so just load on non RTD builds #}
+    {% if not READTHEDOCS %}
+      <script type="text/javascript" src="{{ pathto('_static/js/theme.js', 1) }}"></script>
     {% endif %}
+    <script src="{{ pathto('_static/js/modernizr.min.js', 1) }}"></script>
+    <script type="text/javascript">
```

Comment:
This can probably stay in the footer?

Reply:
According to Paul Irish, in the footer is the way to go for performance, unless you target IE6-8 with the html5shiv: https://github.com/Modernizr/Modernizr/issues/878#issuecomment-41448059
The current Modernizr in the theme does include the shiv, so moving it out of head might break IE6-8. Possible change would be to put the shiv in head and the rest of Modernizr at the end of the file. This relates to #525.

Reply:
This comment was directed towards the script below the modernizr link

Reply:
Whoops. Yes, that should be at the bottom. It is the code that starts the theme JS logic and should be run when the page (or at least the toctree) is complete.

Reply:
Putting JS in the footer is definitely better for performance in general but anything required for rendering should be in the header.

Not that we necessarily have to follow their lead, but Alabaster and the Spinx Basic theme (which Alabaster extends) both put JS in the head.

Reply:
I do think that attempting to have consistency across themes isn't a bad thing though.